### DISCUSSION TITLE
Is it possible to calculate charged molecules with the MACE ASE calculator?

### DISCUSSION THREAD

**Discussion by keceli** [ACEsuit/mace#752] [Q&A]
**Describe the bug**
While I am not sure if MACE-MP models are good for charged molecules, I wanted to give it a try. However, I see that MACE calculator doesn't take the charge into account. How should I set the `charge_key` in the MACE calculator? I tried different keywords (the default 'Qs', charges etc), but none worked so far. I believe this keyword is for the charges in the training set, but I tried it anyways since the calculator didn't make use of the charges set in the `atoms` object. I'd appreciate any help.

**To Reproduce**
The following script should reproduce the problem, you will get the same energy for the neutral and charged molecule.
```python
import ase
from mace.calculators import mace_mp

atoms = ase.build.molecule('H2O')
atoms.set_initial_charges([0,0,0])
atoms.calc = mace_mp(model="medium", dispersion=False, default_dtype="float32", device='cpu', charges_key='initial_charges')
print(atoms.get_potential_energy())
atoms.set_initial_charges([0,0,1])
atoms.calc = mace_mp(model="medium", dispersion=False, default_dtype="float32", device='cpu', charges_key='initial_charges')
print(atoms.get_potential_energy())
```

**Comment by ilyes319** [comment]
While MACE can predict energies and forces on charged molecule, it does not take total charges as input. Models that do that will be available in a few months.

Secondly for molecular systems, I encourage use to use mace-off that is trained specifically for molecular systems.

**Comment by nihil39** [comment]
> While MACE can predict energies and forces on charged molecule, it does not take total charges as input. Models that do that will be available in a few months.
> 
> Secondly for molecular systems, I encourage use to use mace-off that is trained specifically for molecular systems.

Is MACE-OFF better even for inorganic systems like pure water or water with ions?

**Comment by keceli** [comment]
Got it. Thank you @ilyes319.  I had to use mace-mp models, since the charged molecule has a transition metal.

@nihil39 I think MACE-OFF supports only H, C, N, O, P, S, F, Cl, Br, I. See [here](https://mace-docs.readthedocs.io/en/latest/guide/foundation_models.html#mace-off23-transferable-organic-force-fields).
